src/Bullet/bakudan.h

```diff
@@ -17,13 +17,13 @@ typedef struct BakudanWork
     CONTROL        control;
     OBJECT_NO_ROTS field_9C_kmd;
     MATRIX         field_C0_light_mtx[2];
-    MATRIX        *field_100_pMtx;
-    SVECTOR       *field_104;
-    int            field_108;
-    int            field_10C;
-    GV_PAD        *field_110_pPad;
-    int            field_114;
-    int            field_118;
+    MATRIX        *field_100_pMtx;         // (assigned if the c4 is placed on a moving target)
+    SVECTOR       *rotation;               // rotation vector (assigned if the c4 is placed on a moving target)
```

Comment:
This is a position, the matrix is used for the rotation.

Reply:
I got a bit confused (and partially still I am)
Reading more about the gte I saw that the MATRIX struct is defined as
```
typedef struct {
    short m[3][3];  /* 3x3 rotation matrix */
    long  t[3];     /* transfer vector */
} MATRIX;
```
but still this is equivalent to a 4x4 transformation matrix with the last row [0, 0, 0, 1]

```
m00 m01 m02 t1
m10 m11 m10 t2
m20 m21 m22 t3
0   0   0   1
```

so probably field_100_pMtx is not just the rotation matrix but the full transformation matrix

Reply:
There's a seperate vector for position for some reason.